source/developers-guide/plugin-configuration/index.md

```diff
@@ -0,0 +1,685 @@
+be used to retrieve the element from the `$form` object by calling `$form->getElement(NAME)`. Now we the have the element available via 
+`$elementModel`.
+
+Next we can assign the translation to the element:
+```
+//create new translation model
+$translationModel = new \Shopware\Models\Config\ElementTranslation();
+$translationModel->setLabel($snippet);
+$translationModel->setLocale($localeModel);
+ 
+//add the translation to the form element
+$elementModel->addTranslation($translationModel);
+ 
+//done
+```
+Here we create a new `\Shopware\Models\Config\ElementTranslation` Model, which is responsible for the translations 
+of the configuration elements. In this model we set the label translation and the previously loaded 
+`$localeModel`. Last we assign just created `ElementTranslation` model to the element and the translation is automatically saved.
+
+In the __Basic Settings__ our field in English backend now looks like this:
+
+<img src="img/configuration_english.png" alt="Configuration in English backend" class="image-border" />
+
+## Subshop specific plugin configuration
+In this chapter we want to show how to make your configurations subshop specific. To demonstrate this
+we write a little plugin which replaces the Shopware logo with random text.
+
+### Bootstrap
+```
+<?php
+     ...
+ 
+class Shopware_Plugins_Frontend_SwagSubshopVisibility_Bootstrap extends  Shopware_Components_Plugin_Bootstrap
+{
+    public function getLabel()
+    {
+        return 'Plugin Sichtbarkeit';
+    }
+ 
+    public function getInfo()
+    {
+        return array(
+            'label' => $this->getLabel(),
+            'version' => $this->getVersion(),
+            'link' => 'http://www.shopware.de'
+        );
+    }
+ 
+    public function getVersion()
+    {
+        return '1.0.0';
+    }
+ 
+    public function install()
+    {
+        $this->subscribeEvent(
+                'Enlight_Controller_Action_PostDispatch',
+                'onPostDispatch'
+        );
+ 
+        $form = $this->Form();
+        $parent = $this->Forms()->findOneBy(array('name' => 'Frontend'));
+        $form->setParent($parent);
+        $form->setElement(
+            'checkbox', 
+            'show', 
+            array(
+                'label' => 'Plugin Anzeigen',
+                'value' => true,
+                'scope' => Shopware\Models\Config\Element::SCOPE_SHOP
+            )
+        );
+ 
+        return true;
+    }
+ 
+    public function onPostDispatch(Enlight_Event_EventArgs $arguments)
+    {
+        $config = $this->Config();
+        if (empty($config->show)) {
+            return;
+        }
+ 
+        $controller = $arguments->getSubject();
+        $view = $controller->View();
+ 
+        $view->assign('swagSubshopVisibility', 'Test text.');
+ 
+        $view->addTemplateDir($this->Path() . 'Views/');
+        $view->extendsTemplate('frontend/plugins/swag_subshop_visibility/index.tpl');
```

Comment:
extendsTemplate is not valid since SW5.0
delete this line, I add an example for the index.tpl